README.rst

```diff
@@ -86,7 +86,7 @@ Besides the numerical argument, there's two optional arguments.
 * ``ru`` (Russian)
 * ``tr`` (Turkish)
 * ``th`` (Thai)
-* ``vn`` (Vietnamese)
+* ``vi_VN`` (Vietnamese)
 * ``nl`` (Dutch)
```

Comment:
It should not be only `vi` https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=vi?